src/index.test.ts

```diff
@@ -42,6 +48,31 @@ describe("Aracari", () => {
       `An aracari or araçari (US: /ˌɑːrəˈsɑːri/ AR-ə-SAR-ee,[1] UK: /ˌærəˈsɑːri/ ARR-ə-SAR-ee, /-ˈkɑːri/ -⁠KAR-ee)[2] is any of the medium-sized hermosa toucans that, together with the saffron toucanet, make up the genus Pteroglossus.`
     );
   });
+
+  test("replaceText should replace text nodes with passed text nodes", () => {
+    const parentNode = aracari.getTextNode("toucans").parentNode;
```

Comment:
These tests are so helpful in understanding whats going on. Love it.

Reply:
Yeah test help a lot with comprehension of what code is intended to do. 🤗